apps/cowswap-frontend/src/modules/ordersTable/containers/OrderRow/OrderWarning.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { ReactNode } from 'react'
```

Comment:
😬

Reply:
I will clean up all of them afterwards :) 
And I will try to make prettier to remove them automatically